src/gen/docs/apis/VaultApi.md

```diff
@@ -748,27 +748,27 @@ const apideck = new Apideck({
 
 const params = {
   session: {
```

Comment:
shouldn't this be sessionRequest?

Reply:
yeah, looks like the "root" parameter key for request bodies is wrong for many of these examples. They come from `x-typescript-params` in unify, I'll log an issue

Reply:
going to merge this one, already makes the docs more correct than before

Reply:
ok